boards/nrf52840dk/mtd.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2020 Koen Zandberg <koen@bergzand.net>
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nrf52840dk
+ * @{
+ *
+ * @file
+ * @brief       MTD configuration for the nRF52840 DK
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ *
+ * @}
+ */
+
+#ifdef MODULE_MTD
+
+#include "board.h"
+#include "mtd.h"
+#include "mtd_spi_nor.h"
+#include "periph_conf.h"
+#include "timex.h"
+
+static const mtd_spi_nor_params_t _nrf52840dk_nor_params = {
+    .opcode = &mtd_spi_nor_opcode_default,
+    .wait_chip_erase = 50LU * US_PER_SEC,
+    .wait_32k_erase = 240LU *US_PER_MS,
+    .wait_sector_erase = 40LU * US_PER_MS,
+    .wait_4k_erase = 40LU * US_PER_MS,
+    .wait_chip_wake_up = 35LU * US_PER_MS,
```

Comment:
It should now be safe to omit the `LU` postfix, if you feel like this. `US_PER_MS` no has an `LU` postfix, so this would make no difference. But I wouldn't mind also keeping it.

Reply:
If you want to drop the LU, please squash right away. Otherwise, feel free to hit merge ;-)

Reply:
I'm going to keep them in for consistency sake